File: tests/fixtures/go/policies/pii_redaction_before_llm/violating/handler/handler.go
//go:build policyguard_fixture

package handler

import (
	llm "github.com/anthropic/anthropic-sdk-go"
)

func SummarizeUser(userID string) string {
	user := loadUser(userID)
	return llm.Messages.Create(user)
}

func loadUser(userID string) map[string]string {
	return map[string]string{"id": userID, "email": "x@example.com"}
}

File: tests/fixtures/java/policies/pii_redaction_before_llm/violating/com/example/app/Handler.java
package com.example.app;

import com.example.anthropic.Anthropic;

public class Handler {
    private final Anthropic client;

    public Handler(Anthropic client) {
        this.client = client;
    }

    public String summarize(String userId) {
        var user = loadUser(userId);
        return client.messagesCreate(user);
    }

    private java.util.Map<String, String> loadUser(String userId) {
        return java.util.Map.of("id", userId, "email", "x@example.com");
    }
}
